### ISSUE TITLE
Ability to fetch a Card Page from the wiki and update it from the data file

### ISSUE THREAD

**Issue by kushieda-minori** [kushieda-minori/vc_file_grouper#49]
Breakout from #45 

Ability to download page data for a single card and update different sections based on file data. Each section edit would be selectable (i.e. optional):

* [x] Fetch the page from the Wiki
* [x] Base into (Element, Rarity, Symbol, Evolution Accident info)
* [x] Stats
* [x] Exchange info (gold/medal)
* [x] Skills
* [x] Quotes
* [ ] All info for missing evolution (i.e. Rebirth was added, add Stats & Skills & Quote)
* [x] Awakening and Rebirth information
* [ ] upload only missing images (no overwrite) See #48,  Depends on #47
* [x] Write the data back to the wiki: Depends on #47

**Comment by kushieda-minori** [comment]
game is over. won't fix
